h5c/vic/src/vic-webapp/e2e/vch-create-wizard/2-host-affinity.e2e-spec.ts

```diff
@@ -0,0 +1,266 @@
+/*
+ Copyright 2017 VMware, Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import {browser, by, element, protractor, $$} from 'protractor';
+
+import { PROTRACTOR_JASMINE_TIMEOUT } from '../../src/app/testing/jasmine.constants';
+import { VicWebappPage } from '../app.po';
+import {
+  defaultTimeout,
+  sectionSummary,
+  sectionOpsUser,
+  sectionRegistry,
+  sectionSecurity,
+  sectionNetworks,
+  sectionStorage,
+  sectionCompute,
+  modalWizard,
+  dataGridCell,
+  iframeTabs,
+  namePrefix
+} from './common';
+
+describe('VCH Create Wizard - Basic', () => {
```

Comment:
What is difference between this spec and the non host-affinity create spec? They seem very similar. Just seems to be a lot of duplicated code, so trying to see if there is a better way to do it.

Reply:
It's almost the same flow, I have added an automated "Host Affinity" check to activate it on "Compute capacity step", btw I need to test the whole flow now with the host affinity option checked. Since the host affinity is checked during the vch creation, I just take advantage of all the code we have to create a VCH. 
 added a new expect called "should validate the additions of the VCH to the group" if it adds the VCH to the group:

Resuming there is three modification:
1. When we are creating the VCH on the compute capacity windows we have an automated check the host affinity option.
2. After the vch is created we need to navigate to the VCH objects and resource 
3. Check that the vch was added to vch group